features/step_definitions/element_explicit_waiting_steps.rb

```diff
@@ -91,12 +57,21 @@
   expect(@duration).to be_between(0.4, 0.55)
 end
 
+Then('I get a timeout error when waiting for an element with default limit') do
```

Comment:
I get a **Visibility Timeout** error when waiting for an element with default limit

Reply:
So this one, and the others I deliberately cut down to try hit the 80 character limit. However there is an argument that your definitions are better and we could put a rubocop override in place.

Thoughts?